from unittest import TestCase


class Test(TestCase):
    def test_two_sum(self):
        self.fail()
